hw/check_tool_requirements.core

```diff
@@ -8,14 +8,14 @@ description: "Check tool requirements"
 filesets:
   files_check_tool_requirements:
     files:
-      - util/check_tool_requirements.py : { copyto: util/check_tool_requirements.py }
+      - check_tool_requirements.py : { copyto: check_tool_requirements.py }
```

Comment:
This seems weird, copyto yourself?

Reply:
Probably, but I'm not looking to clean up this core right now. There's a listed issue for this that doesn't appear to be closed ( https://github.com/olofk/fusesoc/issues/353 ), and I'll let someone else figure that out. :smile: